src/main/java/com/bitmovin/api/encoding/encodings/Encoding.java

```diff
@@ -35,6 +35,8 @@
 
     private String orgId;
 
+    private List<String> labels;
```

Comment:
does this work? I looked into the Encoding class in the ERS and there it's defined as `List<EncodingLabel>` and not `List<String>`
see: https://github.com/bitmovin/encoding-resource-service/blob/develop/src/main/java/com/bitmovin/api/encodings/persistence/entities/Encoding.java#L58

Reply:
I ran a test and it worked as expected:

```
{
    "requestId": "9024c7ed-2c38-4b5b-b7be-0b828dfd1222",
    "status": "SUCCESS",
    "data": {
        "result": {
            "id": "6725086f-73e8-4916-a116-2aa309244248",
            "name": "Encoding Java Example - Add Labels to Encoding",
            "encoderVersion": "STABLE",
            "createdAt": "2019-01-24T10:09:22Z",
            "modifiedAt": "2019-01-24T10:09:29Z",
            "cloudRegion": "AWS_EU_WEST_1",
            "selectedCloudRegion": "AWS_EU_WEST_1",
            "type": "VOD",
            "status": "ERROR",
            "startedAt": "2019-01-24T10:09:29Z",
            "queuedAt": "2019-01-24T10:09:29Z",
            "runningAt": "2019-01-24T10:10:50Z",
            "errorAt": "2019-01-24T10:10:56Z",
            "progress": 90,
            "labels": [
                "Customer 1"
            ]
        }
    }
}
```

It's also a `List<String>` in the new Open API clients: https://github.com/bitmovin/bitmovin-openapi-clients/blob/develop/java/src/main/java/com/bitmovin/api/model/Encoding.java#L33